### PR TITLE
[14.0][MIG] stock_measuring_device

### PR DESCRIPTION
**Pull request by hailangvn**
This PR depends on below PR.
OCA/product-attribute#908

The column `lngth` is renamed to match with a change in product_packaging_dimension
https://github.com/OCA/product-attribute/commit/24c773cb7c0099bf2f6ac11509e1aaa6781cd03b

### CODE REVIEW

File: stock_measuring_device/wizard/measuring_wizard_line.py [hunk 1]
```diff
@@ -14,9 +14,9 @@ class MeasuringWizardLine(models.TransientModel):
     name = fields.Char("Packaging", readonly=True)
     qty = fields.Float("Quantity")
     max_weight = fields.Float("Weight (kg)", readonly=True)
-    # this is not a typo:
-    # https://github.com/odoo/odoo/issues/41353#issuecomment-568037415
-    lngth = fields.Integer("Length (mm)", readonly=True)
+    # this is renamed to match with a change in product_packaging_dimension
+    # https://github.com/OCA/product-attribute/commit/24c773cb7c0099bf2f6ac11509e1aaa6781cd03b
```
Comment (sebalix) [inline stock_measuring_device/wizard/measuring_wizard_line.py]: You can drop this comment IMO, it's not a valuable information for upcoming migrations (but it's a great info for reviewers if you do it as a comment when you open the PR).

**Review by sebalix — APPROVED** [review]
Pre-approving, small comment

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
can you check pre-commit?

**Comment by sebalix** [comment]
Ah `pre-commit` is failing BTW

**Comment by hailangvn** [comment]
Thanks @simahawk, @sebalix  for your approval. The PR has been updated.

**Pull request merged**